src/android/shakeDetector.java

```diff
@@ -5,33 +5,36 @@
  */
 
 import org.json.JSONArray;
-        import org.json.JSONException;
-        import org.json.JSONObject;
+import org.json.JSONException;
+import org.json.JSONObject;
 
-/*
- * Importing dependencies from the notification plugin
- */
 
-        import de.appplant.cordova.plugin.localnotification.TriggerReceiver;
-        import de.appplant.cordova.plugin.notification.Manager;
+
+import de.appplant.cordova.plugin.localnotification.TriggerReceiver;
+import de.appplant.cordova.plugin.notification.Manager;
 
 /*
  * Importing dependencies from the logger plugin
  */
+import edu.berkeley.eecs.emission.cordova.tracker.Constants;
```

Comment:
Don't introduce a dependency from this plugin to `tracker` (DataCollection) unless you need to.
Are you actually using this anywhere? Most usage appears to be commented out...